README.md

```diff
@@ -17,8 +17,82 @@
 [doi-badge]:           https://zenodo.org/badge/454071954.svg
 [doi-link]:            https://zenodo.org/doi/10.5281/zenodo.10452647
 
-Tools to find evidence for structural variation.
+Tools to gather evidence for structural variation via breakpoint detection.
 
 ## Documentation
 
-Documentation can be found in the [docs folder](docs/01_Introduction.md)
+More detailed documentation can be found in the [docs folder](docs/01_Introduction.md).
+
+## Introduction to the `fgsv` Toolkit
+
+The `fgsv` toolkit contains tools for effective structural variant debugging.
+These tools are not meant to be used as a structural variant calling toolchain in-and-of-itself; instead, it is better to think of `fgsv` as an effective breakpoint detection and structural variant exploration toolkit.
+
+> [!NOTE]
+> When describing structural variation, we use the term **breakpoint** to mean a junction between two loci and the term **breakend** to refer to one of the loci in a breakpoint.
+
+> [!IMPORTANT]
+> All point intervals (1-length) reported by this toolkit are 1-based inclusive from the perspective of the reference sequence unless otherwise documented.
+
+### `SvPileup`
+
+Collates pileups of reads over breakpoint events.
+
+```console
+fgsv SvPileup \
+    --input sample.bam \
+    --output sample.svpileup
+```
+
+The tool [`fgsv SvPileup`](https://github.com/fulcrumgenomics/fgsv/blob/main/docs/tools/SvPileup.md) takes a queryname-grouped BAM file as input and scans each query group (template) of alignments for structural variant evidence.
+For a simple example: a paired-end read may have one alignment per read—one alignment for read 1 and another alignment for read 2 mapped to different reference sequences supporting a putative translocation.
```

Comment:
"may have one alignment per read—one alignment for read 1" doesn't parse for me, can you take a look?

Reply:
Good catch.  Botched sentence due to a Git merge.